CONTRIBUTING.md

```diff
@@ -16,3 +16,12 @@ Deploy a new dev server revision
 ```
 poetry run python cli/main.py remote deploy --image europe-docker.pkg.dev/dbt-server-sbx-f570/dbt-server/dev:avi --service dbt-server-avi
 ```
+
+## Publish the package
+
+Increment the version number in `pyproject.toml`, `cli/version/py`, and `server/version.py`.
```

Comment:
Do we want to centralise this in a single file ? or do they have different lifecycle ?

Reply:
C'est un TODO en effet, j'ai pas encore une bonne idée de comment gérer ça, si tu as une idée je suis assez chaud !